host-configs/tpls.cmake

```diff
@@ -0,0 +1,87 @@
+#
+# Performance portability
+#
+if(EXISTS ${GEOSX_TPL_DIR}/raja)
```

Comment:
and if `GEOSX_TPL_DIR` does not exist?

Reply:
This file is only meant to be included when you define `GEOSX_TPL_DIR`. Once you get to `setupGEOSXThirdParty.cmake` if `RAJA_DIR` isn't defined it'll error out because it's a hard dependency.